libs/ledger-live-common/src/families/icon/js-synchronization.ts

```diff
@@ -0,0 +1,54 @@
+import { encodeAccountId } from "../../account";
+import type { GetAccountShape } from "../../bridge/jsHelpers";
+import { makeSync, makeScanAccounts, mergeOps } from "../../bridge/jsHelpers";
+
+import { getAccount, getOperations } from "./api/node";
+import BigNumber from "bignumber.js";
+import { getDelegation, getIScore, getStake } from "./api/node";
+import { IconResources } from "./types";
+
+const getAccountShape: GetAccountShape = async info => {
+  const { address, initialAccount, currency, derivationMode } = info;
+  const accountId = encodeAccountId({
+    type: "js",
+    version: "2",
+    currencyId: currency.id,
+    xpubOrAddress: address,
+    derivationMode,
+  });
+  const oldOperations = initialAccount?.operations || [];
+  // Needed for incremental synchronisation
+  const skip = 0;
```

Comment:
See previous comment https://github.com/LedgerHQ/ledger-live/pull/1677#discussion_r1422369112

Shouldn't `skip` take a different value to implement incremental sync? Usually it's the latest operation's `blockHeight`, but maybe it should be `operations.length`, or something else?...

Reply:
@heirland

Reply:
skip is still at 0 value ?